lib/routes/njuferret/blog.ts

```diff
@@ -0,0 +1,62 @@
+import { Route } from '@/types';
+import ofetch from '@/utils/ofetch';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+
+export const route: Route = {
+    path: '/blog',
+    categories: ['blog'],
+    example: '/njuferret/blog',
+    parameters: { user: 'Github.io blog name' },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['njuferret.github.io'],
+        },
+    ],
+    name: 'Blogs',
+    maintainers: [],
+    handler,
+};
+
+async function handler() {
+    const promises = [];
+    let pageUrl;
+    const baseUrl = 'https://njuferret.github.io';
+    for (let page = 1; page <= 7; page++) {
+        pageUrl = page === 1 ? baseUrl : `${baseUrl}/page/${page}`;
+        promises.push(ofetch(pageUrl));
+    }
```

Comment:
Only page 1 is enough.

> 很多路由上限10条，无法增大。
> 
> 1. 消耗大量服务器资源。
> 2. 容易被反爬虫。
> 3. 绝大部分过时 feed 没有意义。
>
> _Originally posted by HenryQW in https://github.com/DIYgod/RSSHub/issues/572#issuecomment-416757360_
>
> RSS 是用来获取更新，增大条目无意义
> 
> 特殊需求请在相应模块里自行修改
>
> _Originally posted by DIYgod in https://github.com/DIYgod/RSSHub/issues/572#issuecomment-416802920_
>
> rss 就是用来跟踪内容源的更新内容的，采集全部内容不是rsshub要做的，建议你去找一个爬虫
>
> _Originally posted by yefoenix in https://github.com/DIYgod/RSSHub/issues/7291#issuecomment-809019525_

_Originally posted by TonyRL in https://github.com/DIYgod/RSSHub/discussions/12027#discussioncomment-5184629_

---
> RSS is used to get updates, not history

_Originally posted by DIYgod in https://github.com/DIYgod/RSSHub/issues/3985#issuecomment-586582426_